specs/authorization_mechanisms.table

```diff
@@ -0,0 +1,16 @@
+table_name("authorization_mechanisms")
```

Comment:
Ahh, Linux is failing because these specs need to be gated to darwin. Can you move both of these `.table` files into the `./specs/darwin/` directory?